test/e2e/volumes.go

```diff
@@ -170,6 +171,14 @@ func volumeTestCleanup(client *client.Client, config VolumeTestConfig) {
 	}
 
 	if config.serverImage != "" {
+		if err := waitForPodTerminatedInNamespace(client, config.prefix+"-client", "", config.namespace); !apierrs.IsNotFound(err) {
```

Comment:
Just one more question: the pod was deleted gracefully by kubelet (i.e., none-zero grace period), right? If not, kubelet may take extra time to clean up.